x/magpie/internal/keeper/keeper_test.go

```diff
@@ -1,12 +1,51 @@
 package keeper_test
 
 import (
+	"fmt"
+	"math"
 	"testing"
 
 	"github.com/desmos-labs/desmos/x/magpie/internal/types"
 	"github.com/stretchr/testify/assert"
 )
 
+func TestKeeper_SetDefaultSessionLength(t *testing.T) {
+	tests := []int64{-1, 0, 1, math.MaxInt32}
```

Comment:
why testing invalid lengths such as -1 and 0?

Reply:
Valid observation. I've change the keeper method to return an error if the length is equals or below `0`.